R/reporter-junit.R

```diff
@@ -0,0 +1,163 @@
+#' @include reporter.R
+NULL
+
+classnameOK <- function(text) {
+  gsub("[ \\.]", "_", text)
+}
+
+#' @importFrom xml2 xml_attr<- xml_add_child
+add_broken <- function (parent, type, message) {
```

Comment:
I'd prefer that you not import these functions and instead use fully qualified calls.